src/app/static/src/app/components/header/UserHeader.vue

```diff
@@ -38,13 +39,21 @@
     }
 
     interface Computed {
-        currentLanguage: string
+        helpFilename: string
     }
 
     export default Vue.extend<{}, {}, Computed, Props>({
         computed: {
-            currentLanguage: mapStateProp<RootState, string>(null,
-                (state: RootState) => state.language.toUpperCase())
+            helpFilename: mapStateProp<RootState, string>(null,
+                (state: RootState) => {
+                    var filename = "";
+                    if (state.language == Language.en) {
+                        filename = "Naomi-basic-instructions.pdf";
+                    } else if (state.language == Language.fr) {
+                        filename = "Naomi-instructions-de-base.pdf";
+                    }
+                    return filename;
```

Comment:
This would default to linking to nothing if we introduce another language and forget to add a new helpfile. Which shouldn't happen of course... but since we're falling back to EN everywhere else, maybe it would be better to do that here too.